infrastructure/modules/xc3/main.tf

```diff
@@ -205,6 +186,14 @@ resource "null_resource" "upload_files_on_s3" {
   }
 }
 
+# tflint-ignore: terraform_required_providers
+resource "null_resource" "create_eic_endpoint" {
```

Comment:
@yaseenzafar123 try using terraform data resource instead of null_resource
https://developer.hashicorp.com/terraform/language/resources/terraform-data

Reply:
Added in 8874eaec331e103ff2437399597413b8ffcde25c